security/pfSense-pkg-suricata/files/usr/local/www/suricata/suricata_files.php

```diff
@@ -502,6 +528,10 @@ function build_instance_list() {
 			$file_ip_src .= '<br /><i class="fa fa-search" onclick="javascript:resolve_with_ajax(\'' . $fields['src_ip'] . '\');" title="';
 			$file_ip_src .= gettext("Resolve host via reverse DNS lookup") . "\"  alt=\"Icon Reverse Resolve with DNS\" ";
 			$file_ip_src .= " style=\"cursor: pointer;\"></i>";
+			/* Add GeoIP check icon */
```

Comment:
Consider wrapping the display of the GeoIP check icon with a test to verify that GeoIP2 database download is enabled. The config variable is here:  $config['installedpackages']['suricata']['config'][0]['autogeoipupdate']. If that option is "on", display the GeoIP lookup icon, but if the option is "off", hide the GeoIP lookup icon.